$OpenBSD: patch-lib_MC_MCAsmInfoELF_cpp,v 1.1 2017/07/14 17:14:03 sthen Exp $

- Do not use nonexec stack segment header hints.
- Do not use ident.

Index: lib/MC/MCAsmInfoELF.cpp
--- lib/MC/MCAsmInfoELF.cpp.orig
+++ lib/MC/MCAsmInfoELF.cpp
@@ -27,9 +27,9 @@ MCSection *MCAsmInfoELF::getNonexecutableStackSection(
 }
 
 MCAsmInfoELF::MCAsmInfoELF() {
-  HasIdentDirective = true;
+  HasIdentDirective = false;
   WeakRefDirective = "\t.weak\t";
   PrivateGlobalPrefix = ".L";
   PrivateLabelPrefix = ".L";
-  UsesNonexecutableStackSection = true;
+  UsesNonexecutableStackSection = false;
 }
